src/auth/authSession.ts

```diff
@@ -135,33 +135,34 @@ export function getCoreSession(coreHost: string,
     return Promise.reject('No authRequest provided')
   }
 
-  let payload = null
-  let authRequestObject = null
   try {
-    authRequestObject = decodeToken(authRequest)
+    const authRequestObject = decodeToken(authRequest)
```

Comment:
Core sessions are no longer used, I think we should remove this code in a separate PR.